scripts/k8s-setup/Makefile

```diff
@@ -1,30 +1,42 @@
 # Define the root of the local git clone for the common rules to be able
 # know where they are running from.
-export REPOROOT=${CURDIR}/../
+export REPOROOT=${CURDIR}/../../
 
+# Include the common rules.
+# Use "make help" to see them.
+include ../../.make.defaults
 
-ifneq ($(KFPv2), 1)
-	GREP_V=KFP_V2
+IGNORE := $(shell bash -c "sed -n /=/p  ${REPOROOT}/scripts/k8s-setup/requirements.env | sed 's/=/:=/;s/^/export /'  > makeenv")
+ifeq ($(KFPv2), 1)
+   $(shell bash -c "echo export PIPELINE_VERSION:=$(KFP_v2) >> makeenv")
 else
-	GREP_V=KFP_V1
+   $(shell bash -c "echo export PIPELINE_VERSION:=$(KFP_v1) >> makeenv")
 endif
-IGNORE := $(shell bash -c "sed -n /=/p  ${REPOROOT}/kind/requirements.env | sed 's/=/:=/;/$(GREP_V)/d;s/KFP_V._//;s/^/export /'  > makeenv")
-
 include makeenv
 
-# Include the common rules.
-# Use "make help" to see them.
-include ../.make.defaults
-
-export TOOLS_DIR=${REPOROOT}/scripts/kind/tools
-export KIND_SCRIPTS=${REPOROOT}/scripts/kind
+export TOOLS_DIR=${REPOROOT}/scripts/k8s-setup/tools
+export KIND_SCRIPTS=${REPOROOT}/scripts/k8s-setup
 
 export EXTERNAL_CLUSTER ?= 0
 export DEPLOY_KUBEFLOW ?= 1
 export USE_KFP_MINIO ?= 1
 export POPULATE_TEST_DATA ?= 1
+
 KIND_CLUSTER_NAME ?= dataprep
 
+# The following taken from https://stackoverflow.com/a/65243296/45375
+# Lists all targets and optional help text found in the target.
+# :: rule means we first run the help from the main make.defaults.
+extra-help:	
+	@# Help: Shows additional make macros that can be overridden/used. 
+	@echo ""
+	@echo Overridable macro values include the following:
+	@echo EXTERNAL_CLUSTER - If 0 create a KinD cluster.  EXTERNAL_CLUSTER=$(EXTERNAL_CLUSTER)
+	@echo DEPLOY_KUBEFLOW - If 1 deploy kubeflow pipelines on the cluster. DEPLOY_KUBEFLOW=$(DEPLOY_KUBEFLOW)
+	@echo USE_KFP_MINIO - If 1 and DEPLOY_KUBEFLOW is also 1 then deploy MinIO on the cluster including ingress to access it.  USE_KFP_MINIO=$(USE_KFP_MINIO)
+	@echo POPULATE_TEST_DATA - If 1 populate test data to MinIO.  POPULATE_TEST_DATA=$(POPULATE_TEST_DATA)
```

Comment:
we can use an external Minio Server $MINIO_SERVER, but maybe better to define an S3 server and provide its username and password, see "populate_minio.sh"